templates/seshdash/historical-data.html

```diff
@@ -0,0 +1,90 @@
+{% extends 'seshdash/base.html' %}
+
+{% load staticfiles %}
+
+
+{% block main_content %}
+
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+            {% block header %}
+ 
+                <h2 class="historical-header"> Historical Data </h2>
+
+            {% endblock %}
+            </div>
+        </div>
+
+       <div class="filter-form">
+         <div class="data-sort-container">
+            <p class="sort-help-text"> Select an option to sort by </p>
+            <select class="form-control" id="data-sort">
```

Comment:
can you make this dynamic? These values are hard coded. I would rather they get sent back in the context dict and this is generated dynamically.